src/ladds/particle/Constellation.cpp

```diff
@@ -85,17 +108,19 @@ std::vector<Particle> Constellation::tick() {
       break;
     case Status::inactive:
       // check time and activate if startTime is reached
-      if (simulationTime >= startTime) {
+      if (static_cast<long>(simulationTime) >= startTime) {
         status = Status::active;
+        timeActive = simulationTime - startTime;
+        std::cout << "timeActive for " << constellationName << ": " << timeActive << std::endl;
       } else {
         break;
       }
     case Status::active:
 
```

Comment:
add a comment here what happens in this case. Like the comment for the case above
```cpp
// check time and activate if startTime is reached
```